### PR TITLE
Append addBundledTransactions case to fioAction

### PR DESCRIPTION
**Pull request by andreyvEze**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201602422919538

### CODE REVIEW

File: src/fio/fioEngine.js [hunk 1]
```diff
@@ -269,6 +269,31 @@ export class FioEngine extends CurrencyEngine {
...
+                ({ fio_address: name }) => name === params.fioAddress
+              ).remaining_bundled_tx
+              fioAddress.bundles = updatedBundlesValue
+              this.localDataDirty()
+              return { bundles: updatedBundlesValue, ...res }
+            } catch (e) {
+              //
+            }
+
+            return { bundles: fioAddress.bundles, ...res }
```
Comment (thehobbit85) [inline src/fio/fioEngine.js:295]: Why do you need to refetch the 'getFioAddresses' data. Don't you know how many bundles you added from the previous call already?

**Review by thehobbit85 — CHANGES_REQUESTED** [review]

File: src/fio/fioEngine.js (hunk 1 above)
Comment (andreyvEze) [reply src/fio/fioEngine.js:295]: no, I don't know this information. let me check if this a default value

File: src/fio/fioEngine.js [hunk 2]
```diff
@@ -269,6 +269,31 @@ export class FioEngine extends CurrencyEngine {
             }
             return res
           }
+          case 'addBundledTransactions': {
+            const res = await this.multicastServers(actionName, params)
+            const fioAddress = this.otherData.fioAddresses.find(
+              ({ name }) => name === params.fioAddress
+            )
+
+            if (!fioAddress) return res
```
Comment (andreyvEze) [inline src/fio/fioEngine.js:278]: put this before multicast and of false - throw

Comment (thehobbit85) [reply src/fio/fioEngine.js:278]: Also, don't test for "null" value implicitly.
If you want to check if something is null or undefined, use the correct:
`            if (fioAddress == nulll) return res`

**Comment by thehobbit85** [comment]
Replaced by the clean PR - https://github.com/EdgeApp/edge-currency-accountbased/pull/352
